openmc/deplete/results_list.py

```diff
@@ -105,3 +105,26 @@ def get_eigenvalue(self):
             eigenvalue[i] = result.k[0]
 
         return time, eigenvalue
+
+    def get_depletion_time(self):
+        """Return an array of the average time to deplete a material
+
+        ..note::
+
+            Will have one fewer row than number of other methods,
+            like :meth:`get_eigenvalues`, because no depletion
+            is performed at the final transport stage
+
+        """
+        times = np.empty((len(self) - 1, 1))
```

Comment:
Why not just a single dimension?

Reply:
I think I had, at one time, considered saving the total process time as well as the average. Will use a single dimension